src/routes/(console)/apply-credit/+page.svelte

```diff
@@ -227,24 +227,22 @@
     function getBillingPlan(): Tier | undefined {
         const campaignPlan =
             campaign?.plan && $plansInfo.get(campaign.plan) ? $plansInfo.get(campaign.plan) : null;
-        const orgPlan =
-            selectedOrg?.billingPlan && $plansInfo.get(selectedOrg.billingPlan)
-                ? $plansInfo.get(selectedOrg.billingPlan)
-                : null;
+        const newPlan = page.data.plansInfo.get(billingPlan);
```

Comment:
why the inconsistency between using a `plansInfo` store and one from `page.data`?